packages/client/hmi-client/src/components/models/Model.vue

```diff
@@ -3,8 +3,14 @@
 		<header>
 			<div class="framework">{{ model?.framework }}</div>
 			<div class="header-and-buttons">
-				<h4 v-html="title" />
+				<h4>{{ title }}</h4>
```

Comment:
This needs to be a v-html because of the `highlightSearchTerms()` used on the computed `title`.
`highlightSearchTerms()` returns HTML. This was the only quick way I was able to add highlighting of results during the hackthon.
[Here a good example of what I've done in the Vue.js doc](https://vuejs.org/guide/essentials/template-syntax.html#raw-html)

Reply:
Yeah I suspected there may be some fishy injections going here.  I was mucking around a couple of things out of place thinking they may have upset the linter ... will put it back.